app/views/appropriate_bodies/claim_an_ect/register_ect/show.html.erb

```diff
@@ -4,10 +4,10 @@
 %>
 
 
-<%= govuk_panel(title_text: "You’ve successfully claimed #{PendingInductionSubmissions::Name.new(@pending_induction_submission).full_name}’s induction") %>
+<%= govuk_panel(title_text: "You've successfully claimed #{PendingInductionSubmissions::Name.new(@pending_induction_submission).full_name}'s induction") %>
```

Comment:
We should probably standardise on either curly or straight quotes, we have a mix of both. Maybe we can use a helper method or something for it.